modules/hostap/Kconfig

```diff
@@ -171,6 +175,9 @@ config WIFI_NM_WPA_SUPPLICANT_WPA3
 config WIFI_NM_WPA_SUPPLICANT_AP
 	bool "AP mode support"
 
+config WIFI_NM_HOSTAPD_AP
+	bool "AP mode hostapd support"
```

Comment:
`hostapd based full AP mode` and we can fix `WIFI_NM_WPA_SUPPLICANT_AP` to ` SoftAP mode support based on WPA supplicant`

Reply:
Updated.